### PR TITLE
Task4

### PR DESCRIPTION
**Pull request by YuruMaksau**

### CODE REVIEW

File: Task4/index.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            <li><a href="#a7">1914</a></li>
+            <li><a href="#a8">Section</a></li>
+            <li><a href="#a10">Donate</a></li>
+            <li><a href="https://www.google.com" target="_blank">GOOGLE</a></li>
+        </ul>
+    </nav>
+    <div class="moving">
+<video width="400" height="300" controls="controls">
+    <source src="video/videoplayback%20(1).mp4" type="video/mp4" codecs="avc1.42E01E, mp4a.40.2">
+</video>
```
Comment (pahuta) [inline Task4/index.html]: Please keep the level of nesting tags (no spaces before tag <video>). Also review all your code, it contains a huge amount of the same "mistakes"

Comment (pahuta) [reply Task4/index.html]: Where did you read about attribute `codecs` and what it do? I can't find information about it?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source

Comment (pahuta) [reply Task4/index.html]: Why did you define the value of `controls` attribute? It's not a mistake but there are no sense.
Read more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video)

**Comment by YuruMaksau** [comment]
All comments were fixed.

**Pull request merged**